src/app/jscaip/tests/TableUtils.spec.ts

```diff
@@ -73,6 +73,32 @@ describe('TableUtils', () => {
 
     });
 
+    describe('getLefterMatch', () => {
+
+        it('should return -1 when there is no match', () => {
+            // Given a board without 0
+            const table: Table<number> = [[1, 1, 1]];
+
+            // When asking lefter match of value 0
+            const result: number = TableUtils.getLefterMatch(table, (element: number) => element === 0);
+
+            // Then the result should be -1
+            expect(result).toBe(-1);
+        });
+
+        it('should return index when there is a match', () => {
+            // Given a board with a 0 in index 0
+            const table: Table<number> = [[0, 1, 1]];
+
+            // When asking lefter match of value 0
```

Comment:
when asking the index of value equal to 0

Reply:
mmh, modifié en mieux